Formula/ipython@5.rb

```diff
@@ -99,7 +145,16 @@ class IpythonAT5 < Formula
   end
 
   def install
-    virtualenv_install_with_resources
+    venv = virtualenv_create(libexec)
+
+    %w[appnope backports.shutil_get_terminal_size decorator enum34 ipython_genutils pathlib2 pexpect pickleshare prompt_toolkit ptyprocess Pygments scandir simplegeneric six traitlets wcwidth].each do |r|
```

Comment:
That's a good question; I generated this list by running a diff against the old set, but that's probably not scalable.

The issue is that `ipython` and its dependencies have to be installed before `ipykernel`. In retrospect, either of the following might make more sense:

# Install *all* other `resource`s, `pip_install_and_link buildpath` *then* install `ipykernel`
# Change the `url` to the one corresponding to `ipykernel`

Personally, I'm leaning toward the first option. What do you think?